app/src/main/java/org/groundplatform/android/ui/home/mapcontainer/jobs/DataCollectionEntryPointData.kt

```diff
@@ -28,6 +28,7 @@ data class SelectedLoiSheetData(
   override val canCollectData: Boolean,
   val loi: LocationOfInterest,
   val submissionCount: Int,
+  val canDelete: Boolean = false,
```

Comment:
Please remove the default value as the only usage is setting the value. 

Also, please rename this to `showDeleteLoiButton` to make the intent clear. Same comment throughout `JobMapComposables` class